src/test/java/uk/gov/register/presentation/functional/testSupport/DBSupport.java

```diff
@@ -13,19 +11,20 @@
 import java.sql.SQLException;
 import java.time.Instant;
 import java.util.List;
-import java.util.SortedMap;
-import java.util.TreeMap;
+import java.util.concurrent.atomic.AtomicInteger;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
+import static uk.gov.register.presentation.functional.TestEntry.anEntry;
+
 public class DBSupport {
     public static void main(String[] args) throws IOException, SQLException {
         String filePath = args[1];
 
         DBSupport dbSupport = new DBSupport(TestDAO.get(args[2], "postgres"));
         try (Stream<String> lines = Files.lines(new File(filePath).toPath(), Charset.defaultCharset())) {
-            List<String> collect = lines.collect(Collectors.toList());
-            dbSupport.publishMessages(args[0], collect);
+            AtomicInteger index = new AtomicInteger(0);
```

Comment:
creative use of AtomicInteger, though it feels a little dodgy because it depends on the exact order of processing of the `map()` call. It's a shame there's no `mapIndexed()` function :/